shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -775,7 +775,7 @@ export default {
                   v-clean-html="t('resourceTable.groupLabel.notInANodePool')"
                 />
                 <div
-                  v-if="group.ref && group.ref.template"
+                  v-if="group.ref && group.ref.template && !group.ref.isHarvester"
```

Comment:
I wonder if we could call this `providerSummary` (or something) and have that as a getter that returns the formatted summary.

The default on the machinedeployment model would be to return the HTML as currently formatted unless its Harvester in which case it could return undefined - this if could then be conditional on `providerSummary` being truthy

Reply:
Updated by adding the new `providerSummary` getter, thanks!